modules/internet.sh

```diff
@@ -8,19 +8,27 @@ function internet {
       exit 0
     fi
     echo "false"
-    ;;  
+    ;;
   "reverse")
     if ! nc -w 10 -z 8.8.8.8 53 >/dev/null 1>&2; then
       log_and_exit1 "Error: no internet found"
     fi
     info="$(curl -s ipinfo.io | grep -o '"[^"]*"\s*:\s*"[^"]*"')"
-    echo "$info" | grep -E '"(ip)"'
-    if ! echo "$info" | grep -E 'postal'; then
-      echo "$info" | grep -E '"(city|country)"' | tr '\n' ',' | sed 's/,$/, "postal": "n\/a"\n/' | sed 's/\",\"/\", \"/g'
-    else
-      echo "$info" | grep -E '"(city|country|postal)"' | tr '\n' ',' | sed 's/,$/\n/' | sed 's/\",\"/\", \"/g'
+    ip=$(echo "$info" | grep -e '"ip": "')
+    org=$(echo "$info" | grep -e '"org": "')
+    country=$(echo "$info" | grep -o '"country": "[^;]*')
+    city=$(echo $info | grep -o '"city": "[^;]*' | cut -d '"' -f 4)
+    postal=$(echo $info | grep -o '"postal": "[^;]*' | cut -d '"' -f 4)
+    timezone=$(echo $info | grep -o '"timezone": "[^;]*' | cut -d '"' -f 4)
+
+    if [ -z "$postal" ]; then
+      postal="n/a"
     fi
-    echo "$info" | grep -E '"(org|timezone)"'
+
+    echo "$ip"
+    echo "$org"
+    echo "$country, \"city\": \"$city\", \"postal\": \"$postal\""
+    echo "\"timezone\": \"$timezone\""
```

Comment:
In the original, organization and timezone were at the bottom.  But this is fine 👍